Samples/WebAssembly/Set-AksEdgeWasmRuntime.ps1

```diff
@@ -23,9 +23,9 @@
 
 param(
     [Switch] $enable,
-    [string] $shimVersion = "v0.4.0",
-    [ValidateSet("spin", "slight", "both")]
-    [string] $shimOption = "both"
+    [string] $shimVersion = "v0.9.2",
+    [ValidateSet("spin", "slight", "lunatic", "wws")]
+    [string] $shimOption = "slight"
 )
 
 Write-Host "1. Checking AKS Edge Essentials dependencies" -ForegroundColor Green
```

Comment:
suggestion: Consider adding a powershell transcript for the session so the transactions are logged to file automatically.